tests/test_modeling_common.py

```diff
@@ -249,9 +249,11 @@ def check_save_load(out1, out2):
             # make sure we don't have nans
             out_2 = out2.cpu().numpy()
             out_2[np.isnan(out_2)] = 0
+            out_2 = out_2[~np.isneginf(out_2)]
```

Comment:
It filters out all the image tokens because the image generation module is not released. So we can only generate text tokens.

Hmm, do you think that now we can have false positive test passes for other models? Then I can either overwrite or add a flag

Reply:
Ok. If we know it's for specific tokens then I think it's fine, I just don't want us to paper over something which might be an issue